<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainwindow</class>
 <widget class="QMainWindow" name="Mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EngincutText</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Format</string>
    </property>
    <widget class="QMenu" name="menuFont_size">
     <property name="title">
      <string>Font size</string>
     </property>
     <addaction name="action12"/>
     <addaction name="action14"/>
     <addaction name="action24"/>
    </widget>
    <widget class="QMenu" name="menuFont">
     <property name="title">
      <string>Font...</string>
     </property>
     <addaction name="actionArial"/>
     <addaction name="actionCambria"/>
     <addaction name="actionCalibari"/>
    </widget>
    <addaction name="menuFont"/>
    <addaction name="menuFont_size"/>
    <addaction name="actionWrap_Text"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOption"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Edit tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="actionPrint"/>
   <addaction name="actionslctal"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>View tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShownormal"/>
   <addaction name="actionfullscr"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="images/image.qrc">
     <normaloff>:/Icons/blue-folder-open-document.png</normaloff>:/Icons/blue-folder-open-document.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="statusTip">
    <string>Open new file (Ctrl+O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="images/image.qrc">
     <normaloff>:/Icons/disk--pencil.png</normaloff>:/Icons/disk--pencil.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save (Ctrl+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="statusTip">
    <string>Close (Ctrl+Q)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action12">
   <property name="text">
    <string>12pt</string>
   </property>
  </action>
  <action name="action14">
   <property name="text">
    <string>14pt</string>
   </property>
  </action>
  <action name="action24">
   <property name="text">
    <string>24pt</string>
   </property>
  </action>
  <action name="actionArial">
   <property name="text">
    <string>Arial</string>
   </property>
  </action>
  <action name="actionCambria">
   <property name="text">
    <string>Cambria</string>
   </property>
  </action>
  <action name="actionCalibari">
   <property name="text">
    <string>Calibari</string>
   </property>
  </action>
  <action name="actionWrap_Text">
   <property name="text">
    <string>Wrap Text...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="images/image.qrc">
     <normaloff>:/Icons/disk.png</normaloff>:/Icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="statusTip">
    <string>Save as (Ctrl+shift+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="images/image.qrc">
     <normaloff>:/Icons/printer.png</normaloff>:/Icons/printer.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="statusTip">
    <string>Print  (Ctrl+P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="images/image.qrc">
     <normaloff>:/Icons/document-copy.png</normaloff>:/Icons/document-copy.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="statusTip">
    <string>New Document (Ctrl+N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="images/image.qrc">
     <normaloff>:/Icons/arrow-curve-180-left.png</normaloff>:/Icons/arrow-curve-180-left.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="statusTip">
    <string>Undo messUps! (Ctrl+Z)</string>
   </property>
   <property name="whatsThis">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="images/image.qrc">
     <normaloff>:/Icons/arrow-curve.png</normaloff>:/Icons/arrow-curve.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="statusTip">
    <string>Redo messUps!!! (Ctrl+Y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="images/image.qrc">
     <normaloff>:/Icons/scissors.png</normaloff>:/Icons/scissors.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut</string>
   </property>
   <property name="statusTip">
    <string>Cut selected text (Ctrl+X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="images/image.qrc">
     <normaloff>:/Icons/arrow-continue.png</normaloff>:/Icons/arrow-continue.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
   <property name="statusTip">
    <string>Paste text (Ctrl+V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="images/image.qrc">
     <normaloff>:/Icons/clipboard-paste-document-text.png</normaloff>:/Icons/clipboard-paste-document-text.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy selected text (Ctrl+C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionslctal">
   <property name="icon">
    <iconset resource="images/image.qrc">
     <normaloff>:/Icons/selection-input.png</normaloff>:/Icons/selection-input.png</iconset>
   </property>
   <property name="text">
    <string>selectall</string>
   </property>
   <property name="toolTip">
    <string>select all</string>
   </property>
   <property name="statusTip">
    <string>Select all texts in editor (Ctrl+A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionfullscr">
   <property name="icon">
    <iconset resource="images/image.qrc">
     <normaloff>:/Icons/full screen.png</normaloff>:/Icons/full screen.png</iconset>
   </property>
   <property name="text">
    <string>fullscr</string>
   </property>
   <property name="toolTip">
    <string>Full screen</string>
   </property>
   <property name="statusTip">
    <string>Full screen</string>
   </property>
  </action>
  <action name="actionShownormal">
   <property name="icon">
    <iconset resource="images/image.qrc">
     <normaloff>:/Icons/normalview.png</normaloff>:/Icons/normalview.png</iconset>
   </property>
   <property name="text">
    <string>Shownormal</string>
   </property>
   <property name="toolTip">
    <string>Show normal</string>
   </property>
   <property name="statusTip">
    <string>Normal view (Ctrl+shift+Q)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>plainTextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="images/image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionUndo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRedo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCut</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPaste</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopy</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionslctal</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionfullscr</sender>
   <signal>triggered()</signal>
   <receiver>Mainwindow</receiver>
   <slot>showFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShownormal</sender>
   <signal>triggered()</signal>
   <receiver>Mainwindow</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>Mainwindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
